docs/lib/storage/fragments/js/download.md

```diff
@@ -23,12 +42,37 @@ Send object data for immediate file download:
 ```javascript
 const data = await Storage.get(`filename.txt`, { download: true })
 
-// data is a Blob
+// data.Body is a Blob
 data.Body.text().then(string => { 
   // handle the String data return String 
 })
 ```
 
+You can programmatically download Blobs using JavaScript:
+
+```js
+// https://blog.logrocket.com/programmatic-file-downloads-in-the-browser-9a5186298d5c/
```

Comment:
Is this ok to use ? Does the person that owns the blog understand we would use this? I don't want to recommend a blog link if it leads to the person will remove it. Can we look to have this remove?

Reply:
done!